src/components/ForecastCard/ForecastCard.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <v-card class="pa-2">
+  <v-card class="pa-2 ma-1">
```

Comment:
did this get removed?  Not seeing it

Reply:
Yea. no longer needed now cause there's enough margin on the row